Geometry/CMSCommonData/data/dd4hep/cmsExtendedGeometry2026D65.xml

```diff
@@ -38,6 +38,7 @@
     <Include ref='Geometry/TrackerSimData/data/PhaseII/TiltedTracker404/trackersens.xml'/>
     <Include ref='Geometry/TrackerSimData/data/PhaseII/TiltedTracker700/pixelsens.xml'/>
     <Include ref='Geometry/TrackerRecoData/data/PhaseII/TiltedTracker700/trackerRecoMaterial.xml'/>
+    <Include ref='SimTracker/TrackerMaterialAnalysis/data/trackingMaterialGroups_ForPhaseII.xml'/>
```

Comment:
@vargasa I do not think this update has been appropriate.
Checking e.g. in IB relvals wf. 30234.0 which features the T23 geometry (3D pixels) I see a lot of warnings of the type:
```
Could not process q-name of a DDLogicalPart, reason:
No regex-match for namespace=pixel  name=BModule1Layer1InnerPixelActive

SpecPar selection is:
//pixbar:Phase2PixelBarrel/pixel:Layer1/pixel:RodUnflipped1/pixel:BModule1Layer1/pixel:BModule1Layer1InnerPixelwafer/pixel:BModule1Layer1InnerPixelActive

%MSG
%MSG-e Specific:  (NoModuleName) 19-Aug-2020 07:56:15 CEST pre-events
Could not process q-name of a DDLogicalPart, reason:
No regex-match for namespace=pixel  name=BModule2Layer1InnerPixelActive

SpecPar selection is:
//pixbar:Phase2PixelBarrel/pixel:Layer1/pixel:RodUnflipped1/pixel:BModule2Layer1/pixel:BModule2Layer1InnerPixelwafer/pixel:BModule2Layer1InnerPixelActive

%MSG
%MSG-e Specific:  (NoModuleName) 19-Aug-2020 07:56:15 CEST pre-events
Could not process q-name of a DDLogicalPart, reason:
No regex-match for namespace=pixel  name=BModule3Layer1InnerPixelActive
```

which are present in [CMSSW_11_2_X_2020-08-18-2300](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc7_amd64_gcc820/CMSSW_11_2_X_2020-08-18-2300/pyRelValMatrixLogs/run/30234.0_TTbar_14TeV+2026D65+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal/step3_TTbar_14TeV+2026D65+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal.log), bug e.g. not in [CMSSW_11_2_X_2020-08-16-0000](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc7_amd64_gcc820/CMSSW_11_2_X_2020-08-16-0000/pyRelValMatrixLogs/run/30234.0_TTbar_14TeV+2026D65+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal/step3_TTbar_14TeV+2026D65+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal.log) which does not contain this PR.
Please either clean or fix.

Reply:
Hi @mmusich, I added D64, D63, and D65 on your request https://github.com/cms-sw/cmssw/pull/31112#discussion_r468531119 and it seems that wf was not verified by the bot. Do you see any problem with the D64 and D63? If so I can revert https://github.com/cms-sw/cmssw/pull/31112/commits/336ef8d1852d2edd160ce46a30971977b0f4c685 if not I'll revert changes on D65 only :+1:

Reply:
@vargasa the PR tests run fewer WFs than the IB tests, but additional WFs can be added to PR tests on demand.

Reply:
@vargasa, the others look OK. For the moment you can just revert D65, but if we go this way we should find a way to fix it....